recipe/run_test.py

```diff
@@ -5,3 +5,19 @@
     print(ogr.GetDriver(i).GetName())
 
 import os1_hw
+
+def has_geos():
+    pnt1 = ogr.CreateGeometryFromWkt( 'POINT(10 20)' )
+    pnt2 = ogr.CreateGeometryFromWkt( 'POINT(30 20)' )
+    ogrex = ogr.GetUseExceptions()
+    ogr.DontUseExceptions()
+    hasgeos = pnt1.Union( pnt2 ) is not None
+    if ogrex:
+        ogr.UseExceptions()
+    return hasgeos
+
+if has_geos():
+  print('GDAL has GEOS compiled in')
```

Comment:
Yes, I need GEOS, the test was suggested by @ocefpaf in the other pr. I admit I did not use `assert` much until now.

Do you mean something like:

`assert has_geos(),  'GEOS not available within GDAL'`